cmd/crates/stellar-config/Cargo.toml

```
@@ -0,0 +1,73 @@
+[package]
+name = "stellar-config"
```

I think we should scope this to the CLI. Stellar config is an overloaded term.
```suggestion
name = "stellar-cli-config"
```